src/test/simple_test.tcc

```diff
@@ -32,19 +32,19 @@ template<typename FieldT> void simple_circuit(libsnark::protoboard<FieldT> &pb)
     pb.set_input_sizes(1);
 
     //   g1
-    //  /  \
+    //  /  \ .
```

Comment:
Why did you add `.`?

Reply:
Yeah, this is quite annoying.  One of the compilers was giving warnings about multi-line comments if there was a `\` at the end of a comment.  I could either try to disable that warning, or adjust the form of the comments if you prefer.

Reply:
Afterall, I changed the comments to not use `\` at the end.